### PR TITLE
wla-dx head

### PR DESCRIPTION
**Pull request by endrift**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/wla-dx.rb [hunk 1]
```diff
@@ -14,13 +14,22 @@ class WlaDx < Formula
     sha256 "a78744385e31896de6b34ae3426c6081ada2f28b997f0c5c15555e10bd03d53e" => :mountain_lion
   end
 
+  if build.head?
```
Comment (mistydemeo) [inline Formula/wla-dx.rb]: Would you be able to move this into a `head` block like this?

```ruby
head do
  url "https://github.com/vhelin/wla-dx.git"

  depends_on "cmake => :build"
end

**Review by mistydemeo — CHANGES_REQUESTED** [review]

File: Formula/wla-dx.rb (hunk 1 above)
Comment (mistydemeo) [inline Formula/wla-dx.rb]: Oh, and remove the separate `head` line at the same time.

Comment (endrift) [inline Formula/wla-dx.rb]: I wasn't aware that was possible; I tried to find if it was but I guess I missed it. This is what I mean when I say I don't know Ruby :P

**Comment by endrift** [comment]
Fixed.

**Review by mistydemeo — APPROVED** [review]

**Comment by mistydemeo** [comment]
Thanks!

**Pull request merged**